docs/user-guides/desktop/index.md

```diff
@@ -0,0 +1,82 @@
+# Coder Desktop
+
+Use Coder Desktop to work on your workspaces as though they're on your LAN, no
+port-forwarding required.
+
+## Install Coder Desktop
+
+<div class="tabs">
+
+You can install Coder Desktop on macOS or Windows.
+
+### macOS
+
+1. Use [Homebrew](https://brew.sh/) to install the Coder Desktop from [cask](https://github.com/coder/coder-desktop-macos/tree/main):
```

Comment:
[cask](https://github.com/coder/coder-desktop-macos/tree/main): links are kind of unnecessary as the cask does not live in this repo but https://github.com/coder/homebrew-coder

Reply:
I'll just remove the link entirely for now - it doesn't feel like it'd be a helpful detour for someone to take